### PR TITLE
default precision datetime64.md

### PR DESCRIPTION
**Pull request by OlegGalizin**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/ru/sql-reference/data-types/datetime64.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ sidebar_label: DateTime64
 Позволяет хранить момент времени, который может быть представлен как календарная дата и время, с заданной суб-секундной точностью.
 
 Размер тика (точность, precision): 10<sup>-precision</sup> секунд, где precision - целочисленный параметр. Возможные значения: [ 0 : 9 ].
-Обычно используются - 3 (миллисекунды), 6 (микросекунды), 9 (наносекунды).
```
Comment (vitlibar) [inline docs/ru/sql-reference/data-types/datetime64.md:12]: Could you please edit the english version in the same way?

Comment (OlegGalizin) [reply docs/ru/sql-reference/data-types/datetime64.md:12]: Im sorry. I know english not wery well to write documentation.

The DateTime64 has default precision 3. It is all that i wrote.

**Review by vitlibar — APPROVED** [review]

**Comment by vitlibar** [comment]
@OlegGalizin, please check
https://github.com/ClickHouse/ClickHouse/pull/69892#issuecomment-2370423598

**Comment by OlegGalizin** [comment]
> @OlegGalizin, пожалуйста прими лицензионное соглашение контрибьютора: [#69892 (comment)](https://github.com/ClickHouse/ClickHouse/pull/69892#issuecomment-2370423598)

Сделал

**Comment by robot-ch-test-poll** [comment]
*This is an automated comment for commit cbf8ed093688278506b9990f2e16708b869760f2 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/69892/cbf8ed093688278506b9990f2e16708b869760f2/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/69892/cbf8ed093688278506b9990f2e16708b869760f2/style_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

**Pull request merged**